CMakeLists.txt

```diff
@@ -34,16 +34,16 @@ elseif (ARCH_NAME MATCHES "AMD64")
 endif ()
 
 if ("${OS_NAME}" MATCHES "Linux")
-  set(CMAKE_EXTRA_INCLUDE_FILES unistd.h sys/ptrace.h sys/uio.h sys/user.h)
+  set(CMAKE_EXTRA_INCLUDE_FILES unistd.h sys/ptrace.h sys/user.h)
```

Comment:
I don't think we need this if we specify the headers in the calls to `CHECK_SYMBOL_EXISTS`.

Reply:
I think I agree here. I'll change it. :)